src/main/ts/utils/Comparison.ts

```diff
@@ -0,0 +1,45 @@
+export const enum Comparison {
+  LT = -1,
+  EQ = 0,
+  GT = 1
+}
+
+export const toNumber = (c: Comparison): number => {
```

Comment:
Is this necessary, if it already is a number at runtime?

Reply:
Removed.